misc/Kconfig

```diff
@@ -112,6 +112,13 @@ config CHECK_LINK_MAP
 	help
 	  Run a linker address generation validity checker at the end of the
 	  build.
+
+config VERSION_HEADER
+	bool "Add a version header to binary"
+	default n
```

Comment:
`n` is the default value for `bool` symbols. Leaving out the `default n` generates nicer documentation too.

Reply:
ok. I will do the change